package.json

```
@@ -24,6 +24,7 @@
     "check:es": "eslint '{src,test}/**/*.{js,jsx,ts,tsx}'",
     "check:styles": "stylelint 'src/**/*.scss'",
     "check:format": "prettier . -c",
+    "format": "npx prettier --write .",
```

No need for `npx` here, it's already referred to local `.bin`:

```suggestion
    "format": "prettier --write .",
```

---

Oh, cool. Nice to know.